File: apps/s-2-admin/src/api/reminderMail/ReminderMailWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { DateTimeFilter } from "../../util/DateTimeFilter";

export type ReminderMailWhereInput = {
  id?: IntFilter;
  referenceId?: IntFilter;
  reminderType?: "PENDING_BOOKING_CONFIRMATION";
  elapsedMinutes?: IntFilter;
  createdAt?: DateTimeFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/ReminderMail/Dtos/ReminderMailWhereInput.cs
using CactusDemoDotnet.Core.Enums;

namespace CactusDemoDotnet.APIs.Dtos;

public class ReminderMailWhereInput
{
    public int? Id { get; set; }

    public int? ReferenceId { get; set; }

    public ReminderTypeEnum? ReminderType { get; set; }

    public int? ElapsedMinutes { get; set; }

    public DateTime? CreatedAt { get; set; }
}
